mozci/task.py

```diff
@@ -401,6 +404,37 @@ def classifications(self):
             and any(result.group == self.name and not result.ok for result in t.results)
         ]
 
+    @property
+    def durations(self):
+        def _subtract(tup):
+            return tup[0] - tup[1]
+
+        durations = []
+
+        for task in self.tasks:
+            data = run_query(
+                "group_summary_by_task_id", Namespace(task_id=task.id, group=self.name)
+            )["data"]
+
+            durations.append(
+                sum(
+                    map(
+                        _subtract,
+                        zip(data["result.end_time"], data["result.start_time"]),
```

Comment:
This may be simpler

    durations.append(sum(e-s for e, s in zip(data["result.end_time"], data["result.start_time"]))

ActiveData will accept a `{"format":"list"}`, `{"format":"table"}` and `{"format":"cubel"}` the last one is default, which is a columnar format, and what you are getting.  Sometimes this "columnar format is not useful.  Use `list`, and then your code is easier:

    durations.append(sum(r["end_time"]-r['start_time"] for r in data))

https://github.com/mozilla/ActiveData/blob/dev/docs/jx_tutorial.md#format-clause